### PR TITLE
Check benefits financial support flow tests

### PR DESCRIPTION
**Pull request by Rosa-Fox**
This PR adds outcome tests to the check-benefits-support flow test file. It only tests "happy" paths to ensure desired outcomes are present in the result given the necessary criteria.

### CODE REVIEW

File: test/flows/check_benefits_support_flow_test.rb [hunk 1]
```diff
@@ -455,5 +455,12 @@ class CheckBenefitsSupportFlowTest < ActiveSupport::TestCase
         assert_rendered_outcome text: "Check if you’re eligible for 30 hours free childcare"
       end
     end
+
+    should "render Carer's Allowance when eligible" do
+      add_responses carer_disability_or_health_condition: "yes"
+
+      assert_rendered_outcome text: "Carer's Allowance"
```
Comment (leenagupte) [inline test/flows/check_benefits_support_flow_test.rb:462]: This is unrelated to the changes you are making, but the apostrophe in Carer's Allowance should be a typographic quote, i.e. `’`.  I think this one was missed by accident as the other quotes seem to be correct.

**Review by leenagupte — COMMENTED** [review]
Ed this is looking great.  There are couple of copy and paste errors that need to be fixed but nothing major.

The structure of the YAML data file and the calculator made this really easy to review, so thank you (and Rosa)! 🎉

File: test/flows/check_benefits_support_flow_test.rb (hunk 1 above)
Comment (edwardkerry) [reply test/flows/check_benefits_support_flow_test.rb]: good spot!

**Comment by edwardkerry** [comment]
@leenagupte thanks for catching those errors, I've fixed them up now so ready for a re-review. Thank you!

**Review by leenagupte — APPROVED** [review]
Looks good! 💯

**Pull request merged**